references/cli/system.go

```diff
@@ -348,6 +356,35 @@ func NewSystemDiagnoseCommand(c common.Args) *cobra.Command {
 			}
 			fmt.Println("Result: Clusters are fine~")
 			fmt.Println("------------------------------------------------------")
+			// Diagnoses the link of hub APIServer to cluster-gateway
+			fmt.Println("------------------------------------------------------")
+			fmt.Println("Diagnosing the link of hub APIServer to cluster-gateway...")
+			// Get clientset
+			clientset, err := apiregistration.NewForConfig(config)
+			if err != nil {
+				return err
+			}
+			ctx, cancel := context.WithCancel(context.Background())
+			defer cancel()
+			apiService, err := clientset.APIServices().Get(ctx, APIServiceName, metav1.GetOptions{})
+			if err != nil {
+				return err
+			}
+			for _, condition := range apiService.Status.Conditions {
+				if condition.Type == "Available" {
+					if condition.Status != "True" {
+						cmd.Printf("APIService \"%s\" is not available! \nMessage: %s\n", APIServiceName, condition.Message)
+						return CheckAPIService(ctx, config, apiService)
+					}
+					cmd.Printf("APIService \"%s\" is available!\n", APIServiceName)
+				}
+			}
+			_, err = multicluster.GetClusterGatewayService(context.Background(), k8sClient)
```

Comment:
After you checked the above status, I think you do not need to call this function here anymore.